quickwit/rest-api-tests/scenarii/es_compatibility/_setup.quickwit.yaml

```diff
@@ -36,14 +36,20 @@ json:
     timestamp_field: created_at
     mode: dynamic
     field_mappings:
-        - name: repo.name
-          type: text
-          fast: true
-          indexed: true
-        - name: actor.id
-          type: u64
-          fast: true
-          indexed: true
+        - name: repo
+          type: object
+          field_mappings:
+              - name: name
+                type: text
+                fast: true
+                indexed: true
+        # - name: actor
+        #   type: object
+        #   field_mappings:
+        #       - name: id
+        #         type: u64
+        #         fast: true
+        #         indexed: true
```

Comment:
is this commented on purpose?

Reply:
I can remove it. The original mapping was not doing anything. I tried to fix it, but then I ended up having to fix a lot of tests.

I ended up just removing that mapping.